### PR TITLE
WIP Sound labels

### PR DESCRIPTION
**Pull request by julealgon**

### CODE REVIEW

File: Source/effects.h [hunk 1]
```diff
@@ -742,120 +742,348 @@ enum _sfx_id : int16_t {
...
+	// Let this tome serve as your warning... this is a forbidden place. return from whence you came and your life will be spar1ed
 	PS_WARR3,
+
+	// The creature of flame, by horazon. Should be interesting reading.
 	PS_WARR4,
+
+	// Morta vespa gaia ilumino...
 	PS_WARR5,
+
+	// The liutenant of belial is known as grimspike
```
Comment (Jmgr) [inline Source/effects.h:760]: The lieutenant

Comment (qndel) [reply Source/effects.h:760]: don't bother, it's a draft for a reason, a lot of typos etc. : P

**Comment by AJenbo** [comment]
Maybe we could merge this one part at a time instead of having to have it all done in one big swoop?

**Comment by julealgon** [comment]
> Maybe we could merge this one part at a time instead of having to have it all done in one big swoop?

I believe most of this is now covered by the new enum added for player sounds. This wasn't meant for merging and was just posted to share the texts temporarily (I had added the comments for my own needs at some point before that).

Closing.

**Pull request closed without merging**